mz_bokeh_package/components/confirmation_modal.py

```diff
@@ -0,0 +1,114 @@
+import os
+import logging
+from typing import List
+
+from bokeh.io import curdoc
+from bokeh.models import Widget, Toggle, Button, CustomJS, Column
+
+logger = logging.getLogger(__name__)
+
+BASE_DIR = os.path.dirname(__file__)
+CONFIRMATION_MODAL_MACROS_PATH = os.path.join(BASE_DIR, "../templates/confirmation_modal")
+
+
+class ConfirmationModal:
+    def __init__(self, title: str, content_widgets: List[Widget]):
+        """This class creates a confirmation modal.
+
+        In addition to instantiating this class, it's also required to include the
```

Comment:
@roiweinreb can you add here full instructions on how to use the modal? So also all the code you need to add to the template, css if needed, etc. Alternatively, you can create a MWE.